main/survey/reporting.php

```diff
@@ -17,6 +17,7 @@
 require_once '../inc/global.inc.php';
 require_once 'survey.lib.php';
 $this_section = SECTION_COURSES;
+$cidReq = Security::remove_XSS(api_get_cidreq());
```

Comment:
Because of where cidReq passes through before and also during api_get_cidreq(), you don't need to remove_XSS() on the result of api_get_cidreq().